sos/report/plugins/pacemaker.py

```diff
@@ -55,14 +55,14 @@ def setup_pcs(self):
     def postproc_crm_shell(self):
         self.do_cmd_output_sub(
             "crm configure show",
-            r"passw(\S*)=\S+",
+            r"passw([^\s=]*)=\S+",
```

Comment:
What are all the possible options of strings you try to match here? As in some original version, we were scrubbing `incoming_password=` also - that is IMHO why the `\S*` was (as a wide heuristic) there.

Replacing `\S` by it's opposite `\s` simply sounds strange to me..

Reply:
`\S` is all non-whitespace characters. `[^\s=]` is all non-whitespace, non-equal-sign characters. Is there a more elegant way to do that?

Reply:
See https://github.com/sosreport/sos/pull/2109#issuecomment-641052464 and RHBZ#1845386 for the rationale and the problem I'm trying to solve here.

Reply:
If I understand what Pavel is trying to say, I think he's talking about the distinction between \S and \s, where the former "Matches any character which is not a whitespace character." and the latter "Matches Unicode whitespace characters". So perhaps the regex should be:
[^\S=]
Instead of:

[^\s=]

? As in the regex is perfect, just needs to capitalize the s. Is that what you meant Pavel?

Reply:
@jcastill That's the thing -- it shouldn't be `[^\S=]`.
Previously, we used `\S`, which means "any non-whitespace character."
This patch changes that to `[^\s=]`, which means "any character that is not a member of the set {whitespace, equal sign}". `[^\s=]` is equivalent to "\S that is not an equal sign".

Reply:
ah ok, thanks for the clarification!

Reply:
Ah damn regular expressions - I misinterpreted the `^` /o\. ACK to the patch then, now it makes sense also to me :).

Reply:
No worries; regexes are way too easy to misread :)